ReelTalk/containers/MovieDetailView.js

```diff
@@ -93,6 +93,7 @@ var MovieDetailView = React.createClass({
 var styles = StyleSheet.create({
     scrollView: {
       flex: 1,
+      marginBottom: 50
```

Comment:
We don't. Except the scroll gets cut off by the tab bar. It makes android ugly. So it's just a temporary hack until you guys look at it. I wanted the merge to produce practically the samr exact iOS experience. Remove it and the lolomoRow is cut off partially